docs/src/main/sphinx/release/release-465.md

```diff
@@ -0,0 +1,66 @@
+# Release 465 (13 Nov 2024)
+
+## General
+
+* Add the {func}`cosine_similarity` function for dense vectors. ({issue}`23964`)
+* Add support for reading geometries in [EWKB
+  format](https://en.wikipedia.org/wiki/Well-known_text_representation_of_geometry)
+  with the {func}`ST_GeomFromBinary` function. ({issue}`23824`)
+* Add support for the `ORDER BY` clause in a windowed aggregate function. ({issue}`23929`)
+* Collect larger dynamic filters to improve performance of queries with
+  selective joins. ({issue}`22824`)
+* Fix potential configuration leakage between catalogs using JDBC-based
+  connectors. ({issue}`24058`)
+* Lower probability of queries failing when running with task retries due to
+  running out task descriptors memory. ({issue}`24114`)
+
+## CLI
+
+* Fix quoting of output values when `CSV_UNQUOTED` or `CSV_HEADER_UNQUOTED`
+  format is used. ({issue}`24113`)
+
+## BigQuery connector 
+
+* Fix failure when reading views with `timestamp` columns. ({issue}`24004`)
+
+## Cassandra connector
+
+* Set `cassandra.security=PASSWORD` along with `cassandra.username` and
```

Comment:
@Praveen2112 .. is this now required and if yes.. is it a breaking change or does it automatically add it when a username

Reply:
Are we saying that the `cassandra.security` configuration property is being automatically set if you provide a username and password? Needs to be reworded in some way.

Reply:
It's required now. Not automatically.